MAINTAINERS.md

```diff
@@ -199,38 +199,49 @@ This [repository](https://github.com/powsybl/powsybl-metrix) also provides a C++
 This [repository](https://github.com/powsybl/powsybl-case-server) provides a web service for network cases management.
 The [powsybl-case-datasource](https://github.com/powsybl/powsybl-case-datasource) repository is the client part of this service.
 
-**Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)  
-**Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)  
+**Reviewers:** [etiennehomer](https://github.com/etiennehomer), [EtienneLt](https://github.com/EtienneLt)
+**Committers:** [etiennehomer](https://github.com/etiennehomer), [EtienneLt](https://github.com/EtienneLt)
 
 ### powsybl-network-store-server
 This [repository](https://github.com/powsybl/powsybl-network-store-server) provides a persistent implementation of the IIDM grid model in [PostgreSQL](https://www.postgresql.org/) database. The grid model is exposed as a web service.
-The [powsybl-network-store](https://github.com/powsybl/powsybl-network-store) repository is the client part of this service.
 
-**Reviewers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)  
-**Committers:** [geofjamg](https://github.com/geofjamg), [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs)
+**Reviewers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [EtienneLt](https://github.com/EtienneLt), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)  
+**Committers:** [jonenst](https://github.com/jonenst), [antoinebhs](https://github.com/antoinebhs), [SlimaneAmar](https://github.com/SlimaneAmar), [FranckLecuyer](https://github.com/FranckLecuyer)
```

Comment:
Is it possible to keep it to Jon and Antoine? They are already committers on other repos so we do not add more committers if we stick to them only.

Reply:
We decided at project level that these four people are the new responsibles of the network-store topics as experts on the subject, we are all split in teams now in GridSuite so it would be easier if they could all be commiters